docs/envs.md

```diff
@@ -0,0 +1,93 @@
+`dvc envs`
```

Comment:
is there a difference between `dvc envs` and `dvc environments`?

Reply:
Hmm envs shouldn't be here anymore.